_posts/2025-01-09-struct-decode-intro.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Structured Decoding in vLLM: a gentle introduction"
+author: "Guest Post by BentoML and Red Hat"
+---
+
+**TL/DR**:
+
+- Structured decoding allows precise control over LLM output formats
+- vLLM now supports both outlines and XGrammar backends for structured decoding
+- Recent XGrammar integration brings up to 5x improvement in tokens generations
```

Comment:
You could actually bump this 5x to 20x if you use the updated graphs from office hours https://docs.google.com/presentation/d/1Z78ljqPIg7_KZ7ZAqKO4VDjKG-ytbkbZ/edit#slide=id.g319810a565e_0_444
```suggestion
- Recent XGrammar integration brings up to 5x improvement in time per output token (TPOT) under load
```